vocode/streaming/transcriber/assembly_ai_transcriber.py

```diff
@@ -83,9 +84,8 @@ def send_audio(self, chunk):
             self.buffer = bytearray()
 
     def terminate(self):
-        terminate_msg = str.encode(json.dumps({"terminate_session": True}))
-        self.input_queue.put_nowait(terminate_msg)
         self._ended = True
+        super().terminate
```

Comment:
think this should be `super().terminate()`

Reply:
my bad! fixed :)